### DISCUSSION TITLE
Simplify LaTeX input

### DISCUSSION THREAD

**Discussion by GrimPixel** [freeplane/freeplane#1003] [Help]
To enable it, I need to type `\latex $$`, 9 characters. If it detects whether the first and last characters are both a $, then I just need to type `$$`, 2 characters.

**Comment by dpolivaev** [comment]
Instead of typing ` \latex` you can choose core text format Latex
<img width="345" alt="latex" src="https://user-images.githubusercontent.com/1558052/218125352-375fb22b-c3f4-4be2-8026-2abc74bab76d.png">

There are similar options for notes and details
<img width="354" alt="latex-for-note" src="https://user-images.githubusercontent.com/1558052/218125643-eeafcb08-ceaf-46c6-9902-0bd8e9bc980a.png">

**Reply by GrimPixel** [reply]
This forces all other texts to use the LaTeX font as well as the LaTeX editor.
